src/Cake.Prca.Issues.MsBuild/MsBuildCodeAnalysisProviderAliases.cs

```diff
@@ -0,0 +1,169 @@
+﻿namespace Cake.Prca.Issues.MsBuild
+{
+    using Core;
+    using Core.Annotations;
+    using Core.IO;
+
+    /// <summary>
+    /// Contains functionality related to importing code analysis issues from MSBuild logs to write them to
+    /// pull requests.
+    /// </summary>
+    [CakeAliasCategory(CakeAliasConstants.MainCakeAliasCategory)]
+    [CakeNamespaceImport("Cake.Prca.Issues.MsBuild")]
+    public static class MsBuildCodeAnalysisProviderAliases
+    {
+        /// <summary>
+        /// <para>
+        /// Gets an instance for the MsBuild log format as written by the <code>XmlFileLogger</code> class
+        /// from MSBuild Extension Pack.
+        /// </para>
+        /// <para>
+        /// You can add the logger to the MSBuildSettings like this:
+        /// <code>
+        /// var settings = new MsBuildSettings()
+        ///     .WithLogger(
+        ///         Context.Tools.Resolve("MSBuild.ExtensionPack.Loggers.dll").FullPath,
+        ///         "XmlFileLogger",
+        ///         string.Format(
+        ///             "logfile=\"{0}\";invalidCharReplacement=_;verbosity=Detailed;encoding=UTF-8",
```

Comment:
`invalidCharReplacement=_` is something specific to our logger, this is currently not available in `MSBuild.ExtensionPack`. We should pull that upstream...